x-pack/legacy/plugins/code/public/monaco/override_monaco_styles.scss

```diff
@@ -25,4 +27,18 @@
   span.mtk29 {
     color: $euiColorAccent;
   }
+
+  .diagonal-fill {
+    background: none;
+  }
+
+  .monaco-diff-editor .line-insert,
+  .monaco-diff-editor .char-insert {
+    background-color: rgba(0, 179, 164, 0.1);
```

Comment:
You should use the `lighten` function on an $euiColor variable here, to make its origin and intention more apparent.

Also, the deletion color is much more purple than red (I think you're just using the default `EuiBadge` color?), and if we're overriding one to be green we might as well override the other to be red.